$NetBSD: patch-mozilla-release_old-configure.in,v 1.1 2020/01/22 10:36:24 fox Exp $

* Include include/nss/nss for workaround for neqo-crypto 0.1.6 of firefox-72.0

Taken from www/firefox

--- mozilla-release/old-configure.in.orig	2020-01-03 18:58:03.000000000 +0000
+++ mozilla-release/old-configure.in
@@ -2744,7 +2809,7 @@ AC_SUBST_LIST(MOZ_CAIRO_LIBS)
 AC_SUBST_LIST(MOZ_CAIRO_OSLIBS)
 AC_SUBST(MOZ_TREE_PIXMAN)
 
-BINDGEN_SYSTEM_FLAGS="$_BINDGEN_CFLAGS $NSPR_CFLAGS $NSS_CFLAGS $MOZ_PIXMAN_CFLAGS $MOZ_CAIRO_CFLAGS"
+BINDGEN_SYSTEM_FLAGS="$_BINDGEN_CFLAGS $NSPR_CFLAGS $NSS_CFLAGS $MOZ_PIXMAN_CFLAGS $MOZ_CAIRO_CFLAGS -I${PREFIX}/include/nss/nss"
 AC_SUBST(BINDGEN_SYSTEM_FLAGS)
 BINDGEN_SYSTEM_TOML_FLAGS="$BINDGEN_SYSTEM_FLAGS"
 AC_SUBST_TOML_LIST(BINDGEN_SYSTEM_TOML_FLAGS)
